packages/block-library/src/gallery/dirty-image-options.ios.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { View, InteractionManager } from 'react-native';
+/**
+ * WordPress dependencies
+ */
+import { Picker } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useRef, useEffect } from '@wordpress/element';
+
+const applyToAllImagesOption = {
+	id: 'applyToAllImagesOption',
+	label: __( 'Apply to all images' ),
+	value: 'applyToAllImagesOption',
+};
+
+const options = [ applyToAllImagesOption ];
+
+export default ( { applyImageOptions, cancelImageOptions, isVisible } ) => {
+	const pickerRef = useRef();
+	useEffect( () => {
+		InteractionManager.runAfterInteractions( () => {
+			if ( isVisible && pickerRef.current ) {
+				pickerRef.current.presentPicker();
+			}
+		} );
+	}, [ isVisible ] );
+
+	const onChange = ( value ) => {
+		if ( value === applyToAllImagesOption.value ) {
+			applyImageOptions();
+		}
+	};
+	return (
+		<Picker
+			ref={ pickerRef }
+			options={ options }
+			onChange={ onChange }
+			onCancel={ cancelImageOptions }
+			leftAlign={ true }
+			title={ __( 'Apply to all images in this gallery?' ) }
+		/>
+	);
+};
```

Comment:
`dirty-image-options.ios.js` and `dirty-image-options.android.js` look alike, can we somehow unify both files?